src/hct_mis_api/apps/erp_datahub/models.py

```diff
@@ -2,7 +2,7 @@
 
 
 class FundsCommitment(models.Model):
-    rec_serial_number = models.CharField(max_length=10, blank=True, null=True)
+    rec_serial_number = models.CharField(max_length=10, unique=True)
```

Comment:
I would go for a unique constrain to make it case-insensitive OR the field is unique by case ?